src/components/Cart/CartProducts/CartProducts.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div v-for="item in cart"
```

Comment:
what if product list is empty? there should be something displaying "cart is empty" or whatever